run_benchmarks.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
```

Comment:
should get a small comment explaning the usecase of the file.

Reply:
this file will be deleted